product_category_description/i18n/product_category_description.pot

```diff
@@ -0,0 +1,39 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* product_category_description
+#
```

Comment:
this file would say that it should be deleted.

Reply:
Why? The other modules of this repo have this file

Reply:
nothing, my bad.